pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -86,7 +86,10 @@ func (pa *PodAutoscaler) annotationFloat64(key string) (float64, bool) {
 func (pa *PodAutoscaler) ScaleBounds(asConfig *autoscalerconfig.Config) (int32, int32) {
 	var min int32
 	if pa.Spec.Reachability != ReachabilityUnreachable {
-		min, _ = pa.annotationInt32(autoscaling.MinScaleAnnotationKey)
+		min = asConfig.MinScale
```

Comment:
I'd add a test to the test table for this case (i.e. that when minScale > 1 but unreachable we still return 0) since nothing caught this being one line too high before :)

Reply:
added test

Reply:
not super important at all, but jfwiw it's a little easier to review if you push new commits rather than force pushing a squashed commit (just because that way the reviewer doesn't need to read back through the whole thing), and prow is set up to auto-squash when merging anyway